librz/bin/format/elf/elf.c

```diff
@@ -1415,6 +1415,119 @@ ut64 Elf_(rz_bin_elf_get_section_addr_end)(ELFOBJ *bin, const char *section_name
 	RzBinElfSection *section = get_section_by_name(bin, section_name);
 	return section ? section->rva + section->size : UT64_MAX;
 }
+char* Elf_(rz_bin_elf_section_flag_to_string)(ut64 flag) {
+	char *buff;	
+	buff = rz_str_new("");
+	if(flag & SHF_WRITE) {
+		buff = rz_str_append(buff,"SHF_WRITE "); 
```

Comment:
why the final space after all the strings? The buffer will end up with a final space as well, which is maybe not always desired.

Reply:
One section has multiple flags, and when they get concatenated it becomes unreadable without the space, I can change it to '|' symbol.